source/empty/index.rst

```diff
@@ -275,3 +275,27 @@ In order to use this feature, you will need the `github <https://github.com/PyGi
 You'll have to go to your `github account settings page, in the personnal access token tab <https://github.com/settings/tokens>`_. Click on *generate new token*, give the description you want, and make sure you'll check the *public_repo* box only (no need to check anything else, you can create several access token if you need).
 
 The token will be displayed only once; store it in the ``.gh_token`` file in your plugin directory; and that's all!
+
+Excluding files
++++++++++++++++
+
+You can create a ``.ignore-release`` file at the root of your plugin and list here files and directories you want to explicitely exclude from the release archive. WWrite one `expression <https://docs.python.org/2/library/re.html>`_ per line
+
+::
+
+   .+\.png
+   locales/.+\.po
```

Comment:
If your packaging process requires po files, not sure it is the best example to givez :)